.flowconfig

```diff
@@ -1,5 +1,6 @@
 [ignore]
 node_modules
+.*/node_modules/resolve/test/**
```

Comment:
Because of

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/resolve/test/resolver/malformed_package_json/package.json:2:1

Unexpected end of input, expected the token }
```

Also, according to the [documentation](https://flow.org/en/docs/config/ignore/),

> These regular expressions match against absolute paths. They probably should start with .*

That's why `node_modules` line probably does not work.